python/common/org/python/types/Float.java

```diff
@@ -456,7 +460,7 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
         }
 
         if (other instanceof org.python.types.Int) {
-            long other_val = ((org.python.types.Int) other).value;
+            long other_val = ((org.python.types.Int) other).value; // exponent
```

Comment:
Was this change intended?
Seems a bit out of scope for this PR. Anyway, instead of adding a comment, a more useful change would be to rename the variable to be `exponent`.

Reply:
Oh sorry my bad. I was just reading code and wrote some comments over here and there. I will remove those. Thanks again.